craco.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   webpack: {
     configure: (webpackConfig) => {
       const jsonLinesLoader = {
-        test: require.resolve("./src/data.json"),
+        test: /\.json$/,
```

Comment:
I think we need to use JSON lines loader only for data in `data` folder to avoid possible pitfalls e.g. when we will need to import regular JSON.